packages/x-date-pickers-pro/src/DesktopDateRangePicker/DesktopDateRangePicker.test.tsx

```diff
@@ -26,6 +33,9 @@ const WrappedDesktopDateRangePicker = withPickerControls(DesktopDateRangePicker)
   ),
 });
 
+const getPickerDay = (name: string, picker = '0') =>
+  getByRole(screen.getByMuiTest(`date-range-picker-${picker}`), 'gridcell', { name });
```

Comment:
`getByMuiTest` is deprecated according to https://github.com/mui/material-ui/blob/8d6bae2e39b9c6e28caf49e0fb930ccfba4b711f/test/utils/createRenderer.tsx#L223-L226

Reply:
Refactor selector to avoid the need for `testId` altogether.